src/StripeJsGatewayFactory.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Prometee\PayumStripe;
+
+use Payum\Core\Bridge\Spl\ArrayObject;
+use Prometee\PayumStripe\Action\Api\PayAction;
+use Prometee\PayumStripe\Action\JsCaptureAction;
+
+class StripeJsGatewayFactory extends StripeCheckoutSessionGatewayFactory
```

Comment:
@odiseoteam what do you think about extending an Abstract class `AbstractStripeGatewayFactory` to regroup all common configs and avoid adding unnecessary actions or template configs ? It just a question, no need to change it now.

Reply:
You mean like moving some of the default config to the abstract class and leave the stripecheckoutSession with just the config needed for stripe checkout? That would be great

Reply:
Exactly 😉

Reply:
Totally agreed!